Source/WebKit/UIProcess/API/wpe/WPEWebViewPlatform.cpp

```diff
@@ -396,6 +304,73 @@ Vector<WebKit::WebPlatformTouchPoint> ViewPlatform::touchPointsForEvent(WPEEvent
 }
 #endif // ENABLE(TOUCH_EVENTS)
 
+gboolean ViewPlatform::handleEvent(WPEEvent* event)
```

Comment:
Shouldn't these use bool instead?

Reply:
We need to return a gboolean from the callback and the return value of `handleEvent` is used directly, that's why fore this particular case I used gboolean

Reply:
Good!